File: javascript/arrayLeya/EvenInArray.js
let array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];

let addEven = []
let oddNumbers = []

for(let index = 0; index < array.length; index++) {
	if(array[index] % 2 == 0) {
		addEven.push(array[index])
	} else {
		oddNumbers.push(array[index])
	}
}

console.log("Even Numbers:",addEven);
console.log("Odd Numbers:",oddNumbers);
console.log("Both:",[addEven, oddNumbers]);

File: python/arrayleya/even_in_array.py
array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

add_even = []
odd_numbers = []

for numbers in array:
    if numbers % 2 == 0:
        add_even.append(numbers)
    else:
        odd_numbers.append(numbers)

print("Even Numbers:",add_even)
print("Odd Numbers:",odd_numbers)
print("Both:",[add_even, odd_numbers])
